architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +269,57 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
-      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source repository.
+<3> The the internal registry service IP address, which is the target for pushes to the image stream. This valueis set automatically.
+<3> The SHA identifier that this imagestream tag currently references. Resources that reference this imagestream tag use this identifier.
+<4> The image stream tag.
+
 ====
 
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+* You can trigger Builds and Deployments when new image is pushed to the registry.
+* You can periodically reimport images on a defined schedule. If the source image has changed, that change is picked up and reflected in image stream and 
+trigger the Build and/or Deployment flow.
+* You can share images using fine-grained access control and quickly distribute images across your teams.
+* If the source image changes, the Image Stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+// https://access.redhat.com/solutions/3016781 Caveats around referencing latest tags
+However, a general caveat is to understand what your ImageStreamTag is pointing to, and be careful if it changes what it points to".
```

Comment:
It's not a caveat of using image streams, but docker images in general. IS are only affected with this rule the same way everything else using latest tag.

Reply:
Acutally, IS bypass the problem of using latest by providing you a stable pointer, until the next import.

Reply:
Yeah, this is another place where we really need a glossary to setup the concept of docker tags vs imagestream tags, because they have different characteristics that need to be understood.

(and imagestreamtags that "reference" another imagestreamtag are yet another level of complexity that needs to be explained, and that's where you get into the whole issue of "not using latest because latest could suddenly point to something new" (where it may or may not actually be called latest))

Reply:
@soltysh FYI. The wording "caveat" came from here, which was added as an External Tracker in the original BZ: 
https://access.redhat.com/solutions/3016781